bridge/client/app/_components/ktb-webhook-settings/ktb-webhook-settings.component.spec.ts

```diff
@@ -111,6 +111,16 @@ describe('KtbWebhookSettingsComponent', () => {
     }
   });
 
+  it('should be valid URL when it starts with a secret', () => {
+    const urlControl: AbstractControl = component.webhookConfigForm.get('url') as AbstractControl;
+    const urls = ['{{.secret.gremlin.url}}'];
+
+    for (const url of urls) {
+      urlControl.setValue(url);
+      expect(urlControl.valid).toEqual(true);
+    }
+  });
```

Comment:
Can you please also add tests for some invalid cases? e.g. `{.secret.keptn.url}` or `{{secret.keptn.url}}`